.github/workflows/validate.yaml

```diff
@@ -16,7 +16,7 @@ jobs:
       - name: check file changes
         id: check
         run: |
-          echo "changed_files=$(git diff --name-status -r HEAD^1 HEAD |grep -v D |grep gentx |cut -f2)" >> $GITHUB_OUTPUT
+          echo "changed_files=$(git diff --name-status -r HEAD^1 HEAD |grep -v D |grep "mainnet-dryrun-2/gentx" |cut -f2)" >> $GITHUB_OUTPUT
```

Comment:
So, I have seen both "`dry-run`" and "`dryrun`" in python genesis/ansible scripts. 

Raising awareness that this is going to bite us in the end (even if this particular use here is correct)

Reply:
yes, we have both naming conventions in use but in this case it does not matter much since we are using the same convention inside this repository.